### ISSUE TITLE
Cannot load library with python3.8, but works with python3.6

### ISSUE THREAD

**Issue by lovetheguitar** [MSLNZ/msl-loadlib#32]
Hello again :) 

Thanks for maintaining this great library still. 
I'm wondering if this is something this library can improve/fix or does it have to do with some incompatibility with the newer python version and the dll I try to load.

My code is doing nothing more than:
```python
from msl.loadlib import LoadLibrary

net = LoadLibrary("DLL_name", 'net')
...
```

```bash
 Checking .NET config returned "The useLegacyV2RuntimeActivationPolicy property is enabled" and still cannot load library.
Mixed mode assembly is built against version 'v2.0.50727' of the runtime and cannot be loaded in the 4.0 runtime without additional configuration information.
   at System.Reflection.RuntimeAssembly.nLoadFile(String path, Evidence evidence)
   at System.Reflection.Assembly.LoadFile(String path)
   ```

Appreciate any input. 😄

**Comment by jborbely** [comment]
What version of [pythonnet](https://pypi.org/project/pythonnet/) do you have installed in your Python 3.6 environment and in your Python 3.8 environment?

**Comment by lovetheguitar** [comment]
Both times `pythonnet            2.5.2`.

**Comment by jborbely** [comment]
Can you copy the contents of your `python.exe.config` file to this issue? This file is located in the same folder where your python executable is. To know where the python executable is located, you can run the following from your activated environment

```console
python -c "import sys; print(sys.executable)"
```

**Comment by lovetheguitar** [comment]
Content of `python.exe.config` in the python3.6 environment:
```xml
<?xml version ="1.0"?>
<!--
  Created by the MSL-LoadLib package.

  By default, applications that target the .NET Framework version 4.0+ cannot load assemblies from
  previous .NET Framework versions. You must add and modify the "app".config file and set the
  useLegacyV2RuntimeActivationPolicy property to be "true". For the Python executable this would be
  a python.exe.config (Windows) or python.config (Unix) configuration file.

  For example, Python for .NET (pythonnet, https://pythonnet.github.io/) only works with .NET 4.0+
  and therefore it cannot automatically load a shared library that was compiled with .NET <4.0. If
  you try to load the library and a System.IO.FileNotFoundException is raised then that might
  mean that the library is from .NET <4.0.

  Additionally, the System.IO.FileNotFoundException exception will also be raised if the folder
  that the DLL is located in is not within sys.path, so first make sure that the shared library
  is visible to the Python interpreter.

  See https://support.microsoft.com/kb/2572158 for an overview.

  NOTE: To install pythonnet, run:
  $ pip install pythonnet
-->
<configuration>
    <startup useLegacyV2RuntimeActivationPolicy="true">
        <supportedRuntime version="v4.0" />
        <supportedRuntime version="v2.0.50727" />
    </startup>
</configuration>
```

vs the version copied from the 3.8 environment:

```xml
<?xml version ="1.0"?>
<!--
  Created by the MSL-LoadLib package.

  By default, applications that target the .NET Framework version 4.0+ cannot load assemblies from
  previous .NET Framework versions. You must add and modify the "app".config file and set the
  useLegacyV2RuntimeActivationPolicy property to be "true". For the Python executable this would be
  a python.exe.config (Windows) or python.config (Unix) configuration file.

  For example, Python for .NET (pythonnet, https://pythonnet.github.io/) only works with .NET 4.0+
  and therefore it cannot automatically load a shared library that was compiled with .NET <4.0. If
  you try to load the library and a System.IO.FileNotFoundException is raised then that might
  mean that the library is from .NET <4.0.

  Additionally, the System.IO.FileNotFoundException exception will also be raised if the folder
  that the DLL is located in is not within sys.path, so first make sure that the shared library
  is visible to the Python interpreter.

  See https://support.microsoft.com/kb/2572158 for an overview.

  NOTE: To install pythonnet, run:
  $ pip install pythonnet
-->
<configuration>
    <startup useLegacyV2RuntimeActivationPolicy="true">
        <supportedRuntime version="v4.0" />
        <supportedRuntime version="v2.0.50727" />
    </startup>
</configuration>
```

They appear to be *identical*.

**Comment by jborbely** [comment]
Thanks. Are you able to share the DLL or is it proprietary? For example, could you temporarily create a repository on your github account and then commit the DLL and a minimal python script that causes this issue?

If sharing the DLL is not possible/wanted, then there are a few things you could try:

1. install a different version of `pythonnet`, maybe even one of the [pre-releases](https://pypi.org/project/pythonnet/#history) or build it from [source ](https://github.com/pythonnet/pythonnet) (ideally from a commit that passed the tests)
2. do not load the library using `msl-loadlib`. Try the following to see if `pythonnet` can load it

```python
import os
import sys
import clr

dll_path = r'C:\path\to\library.dll'

head, tail = os.path.split(dll_path)
sys.path.insert(0, head)

filename, _ = os.path.splitext(tail)
clr.AddReference(filename)
```
You can read more information from the `pythonnet` side of things [here](https://github.com/pythonnet/pythonnet/wiki/How-to-call-a-dynamic-library#tutorial-how-to-call-a-dynamic-library). Essentially, all `msl-loadlib` does to load the library is it implements the boilerplate code that you needed to do above, maybe creates the `python.exe.config` file and inspects the objects in the DLL.

If `pythonnet` cannot load the library then `msl-loadlib` cannot load the library.

**Comment by jborbely** [comment]
That `AttributeError: module 'clr' has no attribute 'System'` error that you got indicates that you are probably using `msl-loadlib` &le; v0.7.0. I recommend that you upgrade `msl-loadlib` to the latest release.

Provided that the bitness of the DLL is the same as the bitness of Python then the `LoadLibrary` class can be used. Otherwise you need to implement the [Server32/Client64](https://msl-loadlib.readthedocs.io/en/stable/interprocess_communication.html) procedure.

Yes, the error you got by loading the 32-bit version of the DLL in 64-bit Python implies the 32-bit vs 64-bit issue.

An additional requirement for loading a DLL is for Python to know where all dependences of the DLL, if any, are located. One place for this dependency lookup occurs in the operating systems  `PATH` environment variable. You can permanently add `C:\Program Files\Microsoft Dynamics AX\50\BusinessConnector\Bin` to `PATH` or do it dynamically in your code before calling `LoadLibrary`, for example, 

```python
import os
import sys
from msl.loadlib import LoadLibrary

# there may be dependencies in other folders, add those folders to PATH as well
folder = r'C:\Program Files\Microsoft Dynamics AX\50\BusinessConnector\Bin'
os.environ['PATH'] += os.pathsep + folder

# this is probably not necessary
sys.path.append(folder)

net = LoadLibrary('Microsoft.Dynamics.BusinessConnectorNet.dll', 'net')
```
_Don't forget to restart your terminal/IDE if you modify `PATH`._

Although, I don't know why this dependency issue would suddenly appear in Python 3.8 if the same code works in Python 3.6 (provided that everything else is the same, i.e., same computer, DLL, user account, ...). This dependency issue could be the reason why your `pythonnet_test.py` script is raising the confusing `FileNotFoundException`. Try adding the dependency folder to `os.environ['PATH']` before `clr.AddReference(filename)`.

**Comment by lovetheguitar** [comment]
Sorry, this is getting a bit messy. 
When debugging I tried using the 0.7.0 version, before and now I'm using the latest 0.9.0 version.

I do not know what changed (besides like you said maybe some windows PATH magic), but now pythonnet appears to see the 64bit dll as well, without me manually adding it to the path. 🤷‍♂️ 

Running the `pythonnet_test.py` script with and without the `python.exe.config` file generated by msl-loadlib I get the exact same error (pythonnet-3.0.0a2 msl-loadlib 0.9.0):
```bash
head='C:\\Program Files\\Microsoft Dynamics AX\\50\\BusinessConnector\\Bin', tail='Microsoft.Dynamics.BusinessConnectorNet.dll'
filename='Microsoft.Dynamics.BusinessConnectorNet'
Traceback (most recent call last):
  File ".\pythonnet_test.py", line 13, in <module>
    clr.AddReference(filename)
System.IO.FileLoadException: Mixed mode assembly is built against version 'v2.0.50727' of the runtime and cannot be loaded in the 4.0 runtime without additional configuration information.
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadFrom(String assemblyFile, Evidence securityEvidence, Byte[] hashValue, AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean suppressSecurityChecks, StackCrawlMark& stackMark)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at Python.Runtime.AssemblyManager.LoadAssemblyPath(String name)
   at Python.Runtime.CLRModule.AddReference(String name)
```

Unrelated to my problem I noticed that msl-loadlib seems to fail to catch the pythonnet error and does not generate the `python.exe.config` file with the latest pythonnet pre-release. Instead it directly fails with: 
```bash
File "venv\lib\site-packages\msl\loadlib\load_library.py", line 268, in __init__
    clr.AddReference(os.path.splitext(tail)[0])
System.IO.FileLoadException: Mixed mode assembly is built against version 'v2.0.50727' of the runtime and cannot be loaded in the 4.0 runtime without additional configuration information.
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadFrom(String assemblyFile, Evidence securityEvidence, Byte[] hashValue, AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean suppressSecurityChecks, StackCrawlMark& stackMark)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at Python.Runtime.AssemblyManager.LoadAssemblyPath(String name)
   at Python.Runtime.CLRModule.AddReference(String name)
```

Could any other `additional configuration information` be missing for real? 😆 
I might just keep my python3.6 interpreter in parallel for this.

**Comment by jborbely** [comment]
I'm not sure if there is any additional configuration information missing.

There was a previous issue #6 where someone else was trying to load `Microsoft.Dynamics.BusinessConnectorNet.dll`. The fix to that issue was to make sure that all dependencies were accessible.

The testing repository that was created for that issue is still available. I just cloned it, created a new Python 3.8 x64 environment, did a `pip install pythonnet msl-loadlib` and ran the following script from the cloned folder

```python
import sys
from msl.loadlib import LoadLibrary

mdbc = LoadLibrary('Microsoft.Dynamics.BusinessConnectorNet.dll', 'net')
print(sys.version)
print(mdbc.lib.Microsoft.Dynamics.BusinessConnectorNet)
```
The first time I ran this script I got the ` useLegacyV2RuntimeActivationPolicy` error. The second time I got the following output
```console
3.8.13 (default, Mar 28 2022, 06:59:08) [MSC v.1916 64 bit (AMD64)]
<module 'Microsoft.Dynamics.BusinessConnectorNet'>
```
Here's what `pip list` outputs for this (conda) environment
```console
Package      Version
------------ ---------
certifi      2021.10.8
msl-loadlib  0.9.0
pip          21.2.2
pycparser    2.21
pythonnet    2.5.2
setuptools   61.2.0
wheel        0.37.1
wincertstore 0.2
```
The DLL in this testing repository is probably not the same version as the one you are trying to load, but, see if you can load the DLL from this testing repository.

**Comment by lovetheguitar** [comment]
Sorry for the long pause, as this just hunted me down again, here is my protocol of the same test.

1. `git clone https://gitlab.com/remak_team/open-source/MicrosoftDynamicsConnector_python_wrapper.git`
2. `cd MicrosoftDynamicsConnector_python_wrapper`
3. `python -m venv venv`
4. `.\venv\Scripts\activate`
5. `python -m pip install pythonnet msl-loadlib`
6.  `python .\test_py.py` - test_py.py containing your script above

Output of first run:
```bash
Traceback (most recent call last):
  File "C:\Users\lovetheguitar\Documents\Projects\my_playground\MicrosoftDynamicsConnector_python_wrapper\venv\lib\site-packages\msl\loadlib\load_library.py", line 286, in __init__
    self._assembly = System.Reflection.Assembly.LoadFile(self._path)
nfiguration information.
   at System.Reflection.RuntimeAssembly.nLoadFile(String path, Evidence evidence)
   at System.Reflection.Assembly.LoadFile(String path)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".\test_py.py", line 4, in <module>
    mdbc = LoadLibrary('Microsoft.Dynamics.BusinessConnectorNet.dll', 'net')
  File "C:\Users\lovetheguitar\Documents\Projects\my_playground\MicrosoftDynamicsConnector_python_wrapper\venv\lib\site-packages\msl\loadlib\load_library.py", line 298,    raise OSError(msg)
OSError: The library appears to be from a .NET Framework version < 4.0.
The useLegacyV2RuntimeActivationPolicy property was added to C:\Users\lovetheguitar\Documents\Projects\my_playground\MicrosoftDynamicsConnector_python_wrapper\venv\Scripts\python.exe.config
to fix the "System.IO.FileLoadException: Mixed mode assembly..." error.
Rerun the script, or shutdown and restart the interactive console, to see
if Python can now load the .NET library.
```

Output of the second run:
```bash
Traceback (most recent call last):
  File "C:\Users\lovetheguitar\Documents\Projects\my_playground\MicrosoftDynamicsConnector_python_wrapper\venv\lib\site-packages\msl\loadlib\load_library.py", line 286, in __init__
    self._assembly = System.Reflection.Assembly.LoadFile(self._path)
System.IO.FileLoadException: Mixed mode assembly is built against version 'v2.0.50727' of the runtime and cannot be loaded in the 4.0 runtime without additional configuration information.
   at System.Reflection.RuntimeAssembly.nLoadFile(String path, Evidence evidence)
   at System.Reflection.Assembly.LoadFile(String path)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".\test_py.py", line 4, in <module>
    mdbc = LoadLibrary('Microsoft.Dynamics.BusinessConnectorNet.dll', 'net')
  File "C:\Users\lovetheguitar\Documents\Projects\my_playground\MicrosoftDynamicsConnector_python_wrapper\venv\lib\site-packages\msl\loadlib\load_library.py", line 303, in __init__
    raise OSError(update_msg)
OSError: Checking .NET config returned "The useLegacyV2RuntimeActivationPolicy property is enabled" and still cannot load library.
Mixed mode assembly is built against version 'v2.0.50727' of the runtime and cannot be loaded in the 4.0 runtime without additional configuration information.
   at System.Reflection.RuntimeAssembly.nLoadFile(String path, Evidence evidence)
   at System.Reflection.Assembly.LoadFile(String path)
```

Output of `python -m pip list`:
```bash
Package     Version
----------- -------
msl-loadlib 0.9.0
pip         21.1.1
pycparser   2.21
pythonnet   2.5.2
setuptools  56.0.0
```
Python Version: `Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)] on win32`
Python executable: `/c/Users/lovetheguitar/Documents/Projects/my_playground/MicrosoftDynamicsConnector_python_wrapper/venv/Scripts/python`

I'm not using coda. I installed python via choco. 

😢 Any ideas left?

Thank you for your great efforts. ❤️

**Comment by jborbely** [comment]
I still believe that you have a dependency issue, because we are now executing the same code, loading the same DLL and I cannot reproduce the issue. **What actions have you performed to ensure that 64-bit Python knows where all of the 64-bit DLL dependencies are located?**

To simplify the Python dependencies, I installed the same version of Python and the same Python packages that you have in your environment. I downloaded Python 3.8.10 directly from python.org
```console
>python -VV
Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)]

>python -m pip list
Package     Version
----------- -------
msl-loadlib 0.9.0
pip         21.1.1
pycparser   2.21
pythonnet   2.5.2
setuptools  56.0.0
```
To determine the dependencies of the DLL, I loaded the DLL in an interactive console
```pycon
C:\Users\username\Downloads\MicrosoftDynamicsConnector_python_wrapper-master>python
Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> from msl.loadlib import LoadLibrary
>>> mdbc = LoadLibrary('Microsoft.Dynamics.BusinessConnectorNet.dll', 'net')
>>> mdbc.lib.Microsoft.Dynamics.BusinessConnectorNet
<module 'Microsoft.Dynamics.BusinessConnectorNet'>
>>>
```
While the interactive console was _waiting for my next input_, I opened PowerShell (as an admin) to inspect what additional DLLs are loaded when `Microsoft.Dynamics.BusinessConnectorNet.dll` is loaded. Here's the PowerShell command and the output
```console
PS C:\Windows\system32> Get-Process python | select -ExpandProperty modules | group -Property FileName | select name

Name
----
C:\Users\username\Python38\python.exe
C:\Windows\SYSTEM32\ntdll.dll
C:\Windows\System32\KERNEL32.DLL
C:\Windows\System32\KERNELBASE.dll
C:\Windows\System32\ucrtbase.dll
C:\Users\username\Python38\VCRUNTIME140.dll
C:\Users\username\Python38\python38.dll
C:\Windows\System32\SHLWAPI.dll
C:\Windows\System32\msvcrt.dll
C:\Windows\SYSTEM32\VERSION.dll
C:\Windows\System32\WS2_32.dll
C:\Windows\System32\RPCRT4.dll
C:\Windows\System32\ADVAPI32.dll
C:\Windows\System32\sechost.dll
C:\Windows\SYSTEM32\CRYPTSP.dll
C:\Windows\system32\rsaenh.dll
C:\Windows\System32\bcrypt.dll
C:\Windows\SYSTEM32\CRYPTBASE.dll
C:\Windows\System32\bcryptPrimitives.dll

<NOTE: The above DLLs are loaded once the interactive console starts.>

C:\Users\username\Python38\python3.DLL
C:\Users\username\Python38\DLLs\_socket.pyd
C:\Windows\SYSTEM32\IPHLPAPI.DLL
C:\Users\username\Python38\DLLs\select.pyd
C:\Users\username\Python38\DLLs\_elementtree.pyd
C:\Users\username\Python38\DLLs\pyexpat.pyd
C:\Users\username\Python38\DLLs\_ssl.pyd
C:\Windows\System32\CRYPT32.dll
C:\Users\username\Python38\DLLs\libcrypto-1_1.dll
C:\Windows\System32\USER32.dll
C:\Users\username\Python38\DLLs\libssl-1_1.dll
C:\Windows\System32\win32u.dll
C:\Windows\System32\GDI32.dll
C:\Windows\System32\gdi32full.dll
C:\Windows\System32\msvcp_win.dll
C:\Windows\System32\IMM32.DLL
C:\Users\username\Python38\DLLs\_ctypes.pyd
C:\Windows\System32\ole32.dll
C:\Users\username\Python38\DLLs\libffi-7.dll
C:\Windows\System32\combase.dll
C:\Windows\System32\OLEAUT32.dll
C:\Users\username\Python38\DLLs\_bz2.pyd
C:\Users\username\Python38\DLLs\_lzma.pyd

<NOTE: The above DLLs are loaded once "from msl.loadlib import LoadLibrary" is executed, therefore the following
DLLs are loaded once "mdbc = LoadLibrary('Microsoft.Dynamics.BusinessConnectorNet.dll', 'net')" is executed.
The following are the dependencies of pythonnet -- when "import clr" occurs during the call to LoadLibrary.>

C:\Users\username\Python38\lib\site-packages\clr.pyd
C:\Windows\SYSTEM32\mscoree.dll
C:\Windows\Microsoft.NET\Framework64\v4.0.30319\mscoreei.dll
C:\Windows\SYSTEM32\kernel.appcore.dll
C:\Windows\Microsoft.NET\Framework64\v4.0.30319\clr.dll
C:\Windows\SYSTEM32\VCRUNTIME140_CLR0400.dll
C:\Windows\SYSTEM32\ucrtbase_clr0400.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\mscorlib\baa929dcdedff7e4c150adeda3f53b6a\mscorlib.ni.dll
C:\Windows\Microsoft.NET\Framework64\v4.0.30319\clrjit.dll
C:\Windows\SYSTEM32\urlmon.dll
C:\Windows\SYSTEM32\iertutil.dll
C:\Windows\SYSTEM32\srvcli.dll
C:\Windows\System32\shcore.dll
C:\Windows\SYSTEM32\netutils.dll
C:\Windows\SYSTEM32\SspiCli.dll
C:\Windows\SYSTEM32\windows.storage.dll
C:\Windows\SYSTEM32\Wldp.dll
C:\Windows\SYSTEM32\PROPSYS.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System\d1b08a492d712e019f310913d82efb4d\System.ni.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Core\782dd7dd89e97af687ff0bdfb301ea5f\System.Core.ni.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Configuration\26c7945a20d57e805a32145c8bd1f4f7\System.Configuration.ni.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Xml\048d6dbecc6efc1752a15992767e8482\System.Xml.ni.dll

<NOTE: This is where the dependencies of Microsoft.Dynamics.BusinessConnectorNet.dll start.>

C:\Users\username\Downloads\MicrosoftDynamicsConnector_python_wrapper-master\Microsoft.Dynamics.BusinessConnectorNet.dll
C:\Windows\System32\SHELL32.dll
C:\Windows\WinSxS\amd64_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.9625_none_08e1c10da83fbc83\MSVCR90.dll
C:\Windows\WinSxS\amd64_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.9625_none_08e1c10da83fbc83\MSVCP90.dll
C:\Windows\WinSxS\amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.19041.1706_none_919e8e54cc8d4ca1\gdiplus.dll
C:\Windows\SYSTEM32\ODBC32.dll
C:\Windows\SYSTEM32\SHFOLDER.dll
C:\Windows\SYSTEM32\Secur32.dll
C:\Windows\SYSTEM32\MSIMG32.dll
C:\Windows\SYSTEM32\ACTIVEDS.dll
C:\Windows\SYSTEM32\USERENV.dll
C:\Windows\SYSTEM32\NETAPI32.dll
C:\Windows\SYSTEM32\WINSPOOL.DRV
C:\Windows\WinSxS\amd64_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.9625_none_08e1c10da83fbc83\msvcm90.dll
C:\Windows\SYSTEM32\adsldpc.dll
C:\Windows\SYSTEM32\DPAPI.DLL
C:\Windows\System32\WLDAP32.dll
C:\Windows\SYSTEM32\WKSCLI.DLL
C:\Windows\SYSTEM32\LOGONCLI.DLL
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Web\ce6c34ce3eb3ce9c445ba20b3808279d\System.Web.ni.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Drawing\4dac268a38ead99f93898a086bb8c6f6\System.Drawing.ni.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Comp46f2b404#\325c28fe1fe080a414faa35ed34828b1\System.ComponentModel.DataAnnotations.ni.dll
C:\Windows\assembly\NativeImages_v4.0.30319_64\System.Web.8dc504e4#\88e4b927c5699eda4de7e979ca82ebdc\System.Web.ApplicationServices.ni.dll
```
My recommendation is to verify that all dependencies are available in your 64-bit, Python 3.8 environment. Repeat the steps that I did to determine what DLLs are required, but use your Python 3.6 environment to start (although, since your Python 3.6 is 32-bit that means you are loading a 32-bit version of the DLL and 32-bit dependencies).

**Comment by jborbely** [comment]
Closed due to inactivity.
